build.gradle

```diff
@@ -20,6 +20,17 @@ buildscript {
     }
 }
 
+def getRevision = { ->
+    def stdout = new ByteArrayOutputStream()
```

Comment:
what will happen if this block fails?
will it stop the build process or become `null`?

Reply:
Hmm  it may stop. But thought we don't need to cover the case.

Reply:
Ok, I was just wondering if you want to cover it. LGTM.